### PR TITLE
[40K] Implement Belisarius Cawl

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Belisarius Cawl]]

**Comment by Susucre** [comment]
Oops, another PR did exist #10064

File: Mage.Sets/src/mage/cards/b/BelisariusCawl.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        this.toughness = new MageInt(4);
+
+        // Ultima Founding -- {T}, Tap two untapped artifacts you control: Create a 2/2 white Astartes Warrior creature token with vigilance.
+        Ability ultima = new SimpleActivatedAbility(Zone.BATTLEFIELD, new CreateTokenEffect(new WhiteAstartesWarriorToken()), new TapSourceCost());
+        ultima.addCost(new TapTargetCost(new TargetControlledPermanent(2, 2, artifactFilter, false)));
+        ultima.withFlavorWord("Ultima Founding");
+        this.addAbility(ultima);
+
+        // Master of Machines -- {T}, Tap X untapped creatures you control: Look at the top X cards of your library. You may reveal an artifact card from among them and put it into your hand. Put the rest on the bottom of your library in a random order.
+        Ability master = new SimpleActivatedAbility(Zone.BATTLEFIELD, new BelisariusCawlEffect(), new TapSourceCost());
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/b/BelisariusCawl.java:61]: Is there a reason you can't use `LookLibraryAndPickControllerEffect` here like the other PR?

File: Mage.Sets/src/mage/cards/b/BelisariusCawl.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    private static final FilterControlledPermanent artifactFilter =
+        new FilterControlledPermanent("untapped artifacts you control");
+
+    static {
+        artifactFilter.add(TappedPredicate.UNTAPPED);
+        artifactFilter.add(CardType.ARTIFACT.getPredicate());
+    }
+
+    private static final FilterControlledPermanent creatureFilter =
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/b/BelisariusCawl.java:39]: StaticFilters.FILTER_CONTROLLED_UNTAPPED_CREATURES

File: Mage.Sets/src/mage/cards/b/BelisariusCawl.java (hunk 1 above)
Comment (Susucre) [reply Mage.Sets/src/mage/cards/b/BelisariusCawl.java:61]: I do not think there was an actual reason. The other PR is more succinct, should probably merge that one.

**Pull request closed without merging**